core/templates/dev/head/filters/string-utility-filters/wrap-text-with-ellipsis.pipe.spec.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for the WrapTextWithEllipsisPipe for Oppia.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { WrapTextWithEllipsisPipe } from
+  'filters/string-utility-filters/wrap-text-with-ellipsis.pipe';
+import {NormalizeWhitespacePipe} from
```

Comment:
Please add space before and after bracket.

Reply:
Oh! must have missed that. Done!